### DISCUSSION TITLE
I cannot get the doorbell video streaming working

### DISCUSSION THREAD

**Discussion by xiaofeng-han** [potmat/homebridge-google-nest-sdm#125] [Q&A]
Issue: I cannot get the doorbell video streaming working

Environment
=================
OS: Ubuntu server 22.04.3 LTS (VM on hpyer-V but not in docker container)
Node.js Version: v20.10.0
plugin version: v1.1.20
homebridge installed directly on the ubuntu server, and installed plugin upon.

I go through the setup process and filled in all fields (except video encoder)

I installed the ffmpeg separately. (sudo apt-get install ffmpeg)

Error in log
=================
Could not execute device command: ..."code":404,"errors":[{"message":"WebRtc session not found.","domain":"global","reason":"notFound"}]} ...Front door Doorbell

FFmpeg exited with code: 1 and signal: null (Error) Front door Doorbell

**Comment by potmat** [comment]
Well this is a new one.

Can you enable debug mode and post more logs from when the error happens?
